LICENSE

```diff
@@ -1,6 +1,7 @@
     MIT License
 
-    Copyright (c) Microsoft Corporation. All rights reserved.
+    Copyright (c) 2018-present Microsoft Corporation.
```

Comment:
is this `present` correct? or should this be `2018-2023`?

Reply:
Not 2023, because otherwise we will need to update it every year. In PyTorch they put present https://github.com/pytorch/pytorch/blob/main/LICENSE#L15. I have also seen 2018- or 2018.